examples/jmpTestExample.s

```diff
@@ -0,0 +1,5 @@
+; TinyRAM V=1.00 W=16 K=16”
+jmp 4
+answer 1
+answer 0
+Expected result: 0
```

Comment:
It's a comment: either put ';' before or remove the comment

Reply:
This instance is resolved.